packages/@aws-cdk-testing/cli-integ/tests/tool-integrations/amplify.integtest.ts

```diff
@@ -28,9 +28,12 @@ integTest('amplify integration', withToolContext(async (context) => {
   await shell.shell(['npm', 'create', '-y', 'amplify']);
   await shell.shell(['npx', 'ampx', 'configure', 'telemetry', 'disable']);
 
+  const awsCreds = context.aws.identityEnv();
+
   await shell.shell(['npx', 'ampx', 'sandbox', '--once'], {
     modEnv: {
       AWS_REGION: context.aws.region,
+      ...awsCreds
```

Comment:
This was just missing because this test is in a different suite, which I wasn't testing against.